staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -381,15 +408,17 @@ func (e *Store) Create(ctx context.Context, obj runtime.Object, createValidation
 		}
 		return nil, err
 	}
+	// The operation has succeeded.  Call the cleanup function if there is one,
+	// and then make sure the defer doesn't call it again.
+	if cleanup != nil {
+		cleanup(true)
```

Comment:
if `cleanup()` panics here, the current code will call it again in the defer (applies in Update paths as well)

Reply:
Copy it to a temp var and call that?

Reply:
fixed in next push